src/main/java/persistence/bootstrap/EntityLoaderBinder.java

```diff
@@ -0,0 +1,49 @@
+package persistence.bootstrap;
+
+import jdbc.JdbcTemplate;
+import jdbc.RowMapper;
+import persistence.entity.CollectionLoader;
+import persistence.entity.EntityLoader;
+import persistence.entity.proxy.ProxyFactory;
+import persistence.meta.EntityTable;
+import persistence.sql.dml.DmlQueries;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class EntityLoaderBinder {
+    private final Map<String, EntityLoader> entityLoaderRegistry = new HashMap<>();
+
+    public EntityLoaderBinder(List<Class<?>> entityTypes, EntityTableBinder entityTableBinder,
+                              CollectionLoaderBinder collectionLoaderBinder, RowMapperBinder rowMapperBinder, JdbcTemplate jdbcTemplate,
+                              DmlQueries dmlQueries, ProxyFactory proxyFactory) {
+        for (Class<?> entityType : entityTypes) {
+            final EntityTable entityTable = entityTableBinder.getEntityTable(entityType);
+            final EntityLoader entityLoader =
+                    getEntityLoader(entityTableBinder, collectionLoaderBinder, rowMapperBinder, jdbcTemplate, dmlQueries, proxyFactory, entityType, entityTable);
+            entityLoaderRegistry.put(entityType.getTypeName(), entityLoader);
+        }
+    }
+
+    public EntityLoader getEntityLoader(Class<?> entityType) {
+        return entityLoaderRegistry.get(entityType.getName());
+    }
+
+    private EntityLoader getEntityLoader(EntityTableBinder entityTableBinder, CollectionLoaderBinder collectionLoaderBinder,
+                                         RowMapperBinder rowMapperBinder, JdbcTemplate jdbcTemplate, DmlQueries dmlQueries,
+                                         ProxyFactory proxyFactory, Class<?> entityType, EntityTable entityTable) {
```

Comment:
```suggestion
    private EntityLoader getEntityLoader(EntityTableBinder entityTableBinder, CollectionLoaderBinder collectionLoaderBinder,
                                         RowMapperBinder rowMapperBinder, JdbcTemplate jdbcTemplate, DmlQueries dmlQueries,
                                         ProxyFactory proxyFactory, Class<?> entityType) {
        final EntityTable entityTable = entityTableBinder.getEntityTable(entityType);
```

파라미터가 좀 많다고 느껴지는데, 마지막 `EntityTable` 하나는 줄여볼 수 있을 것 같아요. :)